control/freqplot.py

```diff
@@ -131,7 +132,18 @@ def bode_plot(syslist, omega=None,
     grid : bool
         If True, plot grid lines on gain and phase plots.  Default is set by
         `config.defaults['bode.grid']`.
-
+    initial_phase : float
+        Set the reference phase to use for the lowest frequency.  If set, the
+        initial phase of the Bode plot will be set to the value closest to the
+        value specified.  Default is 180 if wrap_phase is False, 0 if
+        wrap_phase is True.
```

Comment:
Mention, that this is in degrees or radians depending on the deg parameter?

Reply:
Fixed.